tests/dates/test_DateTimeInfoExtractor.py

```diff
@@ -391,3 +394,14 @@ def test_correct_df_returned_datetime_input(self):
             expected,
             "incorrect dataframe returned",
         )
+
+    def test_is_serialisable(self):
```

Comment:
I have been less on top of the latest testing changes so more of a question for my own understanding. If we are adding the generic test for serialization to the base transformer, do we need this test here too?

Reply:
Missed the note in the PR explaining why this was the case, will close this comment